google-cloud-storage/src/main/java/com/google/cloud/storage/otel/OpenTelemetryInstance.java

```diff
@@ -42,19 +41,18 @@ class OpenTelemetryInstance implements OpenTelemetryTraceUtil {
 
   private final String transport;
 
-  public OpenTelemetryInstance(StorageOptions storageOptions) {
+  OpenTelemetryInstance(StorageOptions storageOptions) {
     this.storageOptions = storageOptions;
     this.openTelemetry = storageOptions.getOpenTelemetrySdk();
-    this.tracer =
-        openTelemetry.getTracer(LIBRARY_NAME, GaxProperties.getLibraryVersion(this.getClass()));
+    this.tracer = openTelemetry.getTracer(LIBRARY_NAME, storageOptions.getLibraryVersion());
```

Comment:
Switch to using the logic from storageOptions rather than gax. Our version is more stable to repackaging, and is what is used for headers and such. We will want to keep them in sync.